exarl/workflows/workflow_vault/sync_learner.py

```diff
@@ -588,6 +598,12 @@ def episode_round(self, exalearner):
             if nepisodes < ExaComm.agent_comm.size - ExaComm.num_learners:
                 nepisodes = ExaComm.agent_comm.size - ExaComm.num_learners
 
+        # Add episode and step counters to env
+        if not hasattr(exalearner.env, "workflow_episode"):
+            setattr(exalearner.env.env, "workflow_episode", 0)
```

Comment:
May be I am not understanding this correctly, but every environment doesn't have an `env` attribute.

Reply:
So every env does have... There is a mistake on like 601 though.  The env is type ExaEnv, which is wrapping an env.  It took me all morning to track that down.  I think it is worthwhile to ask why we are using ExaEnv.  To me I can't find anywhere we are using it...

Reply:
If we want to get rid of ExaEnv lets do it in another PR.  Marking this as resolved.